src/commands/app/pack.js

```diff
@@ -250,6 +251,8 @@ class Pack extends BaseCommand {
     return files
       .map(file => file.path)
       .filter(file => !filesToExclude.includes(file))
+      .filter(junk.not) // no junk files like .DS_Store
+      .filter((file) => !/^\..*/.test(file)) // no files that start with a '.'
```

Comment:
Should we stdout/document this behavior somewhere?

Reply:
good idea - maybe as a debug log. Maybe stdout for files we *include* would be better instead? (like when running npm pack in a npm package folder)

Reply:
added in https://github.com/adobe/aio-cli-plugin-app/pull/734/commits/834b482413b787b1ae79860fe8012c9614e64996:
1. debug logging of omitted hidden dotfiles
2. debug logging of omitted junk files
3. stdout logging of files that will be packed